src/humancellatlas/data/metadata/__init__.py

```diff
@@ -33,7 +36,8 @@ def from_json(cls, json: JSON, **kwargs):
     def __init__(self, json: JSON) -> None:
         super().__init__()
         self.json = json
-        self.document_id = UUID4(json['hca_ingest']['document_id'])
+        provenance = json.get('hca_ingest') or json['provenance']
```

Comment:
This was the litmus test for me. Will I be able to maintain the same external API while making it work for the old v5 and the new vx bundles? 

At some point we need to remove the v5 support because there will never again be bundles using it after we reingest everything with vx.